pyteal/compiler/compiler.py

```diff
@@ -308,6 +308,16 @@ def compileTeal(
             )
         teal = createConstantBlocks(teal)
 
+    def fmt_cmt(teal_line: str) -> str:
+        try:
+            idx = teal_line.index("// pyteal-src")
+            if idx <= 0:
+                return teal_line
+        except Exception:
+            return teal_line
+
+        return teal_line[:idx].strip(" \t") + "\t" * 2 + teal_line[idx:]
+
     lines = ["#pragma version {}".format(version)]
-    lines += [i.assemble() for i in teal]
+    lines += [fmt_cmt(i.assemble()) for i in teal]
```

Comment:
nit: I am understanding correctly, I think this would enforce all the line to come with trace comment.

Do you think it is reasonable to add a keyword argument in `compileTeal` to activate/deactivate trace in generated Teal?

Reply:
yes! a debug flag would be great addition for this and other thigns as well i think

Reply:
let's consider this: just like `optimizeOptions`, we should also consider something like `debugOptions`, does that sound plausible to you?

Reply:
added `debug` as a simple boolean for now